OrbitCore/Path.cpp

```diff
@@ -126,63 +117,63 @@ std::string Path::GetDllName(bool a_Is64Bit) {
   return a_Is64Bit ? "Orbit64.dll" : "Orbit32.dll";
 }
 
+static std::string GetConfigPath() {
```

Comment:
Maybe something like `CreateAndGetConfigPath` to make it clear that this function does have side effects.

Reply:
Renamed the function, thanks!